step_by_step_contribute.md

```diff
@@ -113,7 +113,7 @@ Edit some code on the `new-branch-name` branch and save your changes to fix the
 
 4. Add new command for dmctl in [root commnd](dm/ctl/ctl.go)
 
-    ```
+    ```go
```

Comment:
There are both `go` and `golang` in this doc, please check which one is supported?